_data-prepper/pipelines/configuration/sources/dynamo-db.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: default
+title: dynamodb
+parent: Sources
+grand_parent: Pipelines
+nav_order: 3
+---
+
+# dynamodb
+
+The `dynamodb` source enables change data capture (CDC) on [DynamoDB](https://aws.amazon.com/dynamodb/) tables. It can receive table events such as `create`, `update`, or `delete` via DynamoDB streams and supports initial snapshots using [point-in-time recovery (PITR)](https://aws.amazon.com/dynamodb/pitr/).
```

Comment:
```suggestion
The `dynamodb` source enables change data capture (CDC) on [DynamoDB](https://aws.amazon.com/dynamodb/) tables. It can receive table events such as `create`, `update`, or `delete` using DynamoDB streams and supports initial snapshots using [point-in-time recovery (PITR)](https://aws.amazon.com/dynamodb/pitr/).
```